src/theme/variables.scss

```diff
@@ -38,7 +38,8 @@ $colors: (
   secondary:  #32db64,
   danger:     #f53d3d,
   light:      #f4f4f4,
-  dark:       #222
+  dark:       #222,
+  translucent: rgba(0, 0, 0, 0.35)
```

Comment:
translucent is not really a color :) also we should use only one color definition